### PR TITLE
Update action.yml

### PR DESCRIPTION
**Pull request by microbit-mark**
Add failing check. Should close https://github.com/microbit-foundation/dev-docs/issues/153

### CODE REVIEW

**Comment by microbit-mark** [comment]
@microbit-carlos can you give this a quick review?

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -20,3 +20,5 @@ jobs:
         title: Link Checker Report
         content-filepath: ./link-checker/out.md
         labels: links
+    - name: Fail if there were link errors
+      run: exit ${{ steps.lc.outputs.exit_code }}
```
Comment (microbit-carlos) [inline .github/workflows/action.yml:24]: Also need to set the `lc` id to the `peter-evans/link-checker@v1` step: https://github.com/peter-evans/link-checker#creating-a-failing-check-for-link-errors

Comment (microbit-mark) [reply .github/workflows/action.yml:25]: Set in https://github.com/microbit-foundation/dev-docs/pull/230/commits/f89315c59822ca5911c06a082425b162e10705d7

**Review by microbit-carlos — APPROVED** [review]
LGTM 👍

**Pull request merged**